### DISCUSSION TITLE
SassWarning: Deprecation $weight: Passing a number without unit % (0) is deprecated.

### DISCUSSION THREAD

**Discussion by davideluque** [twbs/bootstrap#38550] [Q&A]
Despite upgrading Bootstrap to 5.2.3, the deprecation warning persists for me 🤕 

```
SassWarning: Deprecation $weight: Passing a number without unit % (0) is deprecated.

To preserve current behavior: $weight * 1%

More info: https://sass-lang.com/d/function-units

node_modules/bootstrap/scss/_functions.scss 207:11           tint-color()
node_modules/bootstrap/scss/_functions.scss 217:57           shift-color()
node_modules/bootstrap/scss/_list-group.scss 185:30          @import
node_modules/bootstrap/scss/bootstrap.scss 35:9              @import
frontend/stylesheets/globals/_bootstrap_overrides.scss 72:9  @import
frontend/stylesheets/globals/_all.scss 6:9                   @import
frontend/stylesheets/application.scss 13:9                   root stylesheet
```

```
SassWarning: Deprecation $weight: Passing a number without unit % (0) is deprecated.

To preserve current behavior: $weight * 1%

More info: https://sass-lang.com/d/function-units

node_modules/bootstrap/scss/_list-group.scss 187:32          @import
node_modules/bootstrap/scss/bootstrap.scss 35:9              @import
frontend/stylesheets/globals/_bootstrap_overrides.scss 72:9  @import
frontend/stylesheets/globals/_all.scss 6:9                   @import
frontend/stylesheets/application.scss 13:9                   root stylesheet
```

```
bootstrap@5.2.3:
  version "5.2.3"
  resolved "https://registry.yarnpkg.com/bootstrap/-/bootstrap-5.2.3.tgz#54739f4414de121b9785c5da3c87b37ff008322b"
  integrity sha512-cEKPM+fwb3cT8NzQZYEu4HilJ3anCrWqh3CHAok1p9jXqMPsPTBhU25fBckEJHJ/p+tTxTFTsFQGM+gaHpi3QQ==

"sass": "^1.62.1",
"sass-loader": "^13.2.2",
```

**Comment by julien-deramond** [comment]
Just to be sure I've created two StackBlitz based on [our Webpack example](https://github.com/twbs/examples/tree/main/webpack/). `sass` and `sass-loader` deps have the same versions as you.

The first one forces `"bootstrap": "5.2.2"` in the `package.json`: https://stackblitz.com/edit/github-ftzif2.
When you load this project, there's a compilation problem.

The second one forces `"bootstrap": "5.2.3"` in the `package.json`: https://stackblitz.com/edit/github-ftzif2-gbpk5g?file=package.json
When the project is loaded, no compilation problem.

Would you be able to reproduce your issue with Bootstrap 5.2.3 in a StackBlitz project?

**Reply by davideluque** [reply]
Strangely, my issue is not in the opaque() function like users have been reporting. I have it in the function called tint-color.

**Reply by davideluque** [reply]
I am comparing my implementation with the stackblitz example

**Comment by davideluque** [comment]
@julien-deramond In my bootstrap overrides, I have:

$list-group-item-color-scale: 0;

Commenting out this line resolves the error. However, I couldn't reproduce the issue on StackBlitz.

**Comment by davideluque** [comment]
Adding a percentage to the value, like "0%", resolves the error. I will close this and re-open if something else comes up

Thank you for the help Julien!

**Reply by julien-deramond** [reply]
Ah yes, `$list-group-item-color-scale` in Bootstrap v5.2.3 is used alongside `shift-color` which uses `tint-color`. Indeed `0` not being exactly `0%`, it can cause errors like the one you had. Tricky use cases for some Sass vars that can be customized.
Glad you found a solution. Don't hesitate if you find other issues.
